src/components/ha-form/ha-form.ts

```diff
@@ -106,6 +113,12 @@ export class HaForm extends LitElement implements HaFormElement {
                     ${this._computeError(error, item)}
                   </ha-alert>
                 `
+              : warning
```

Comment:
I felt showing a red error was not fitting here, so warning capabilities were added to this component.